pulp_smash/tests/rpm/api_v2/test_sync_publish.py

```diff
@@ -265,6 +266,9 @@ def setUpClass(cls):
             cls.sync_repo,
         )
         cls.task_bodies = tuple(poll_spawned_tasks(cls.cfg, report))
+        cls.repos_iter = tuple((
+            get_repo(cls.cfg, attrs['_href']) for attrs in cls.attrs_iter
+        ))
```

Comment:
Basically, `cls.attrs_iter` and `cls.repos_iter` are duplicate variables. The change I've suggested merges them.